slang-kotlin/src/main/java/com/sonarsource/slang/kotlin/KotlinSensor.java

```diff
@@ -63,7 +63,14 @@ public void execute(SensorContext sensorContext) {
 
   private void analyseFiles(SensorContext sensorContext, Iterable<InputFile> inputFiles, List<TreeVisitor<InputFileContext>> visitors) {
     for (InputFile inputFile : inputFiles) {
-      analyseFile(sensorContext, inputFile, visitors);
+      try {
+        analyseFile(sensorContext, inputFile, visitors);
+      } catch (IllegalStateException e) {
+        // FIXME TBD
+      } catch (Throwable e) {
+        // FIXME TBD
```

Comment:
Maybe just a comment on why we did it like this for now (assert errors following kotlin lexer exception), so that we do not forget.